### PR TITLE
[CLOSED] 5.45x39mm and 9x18mm cartridge migration

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
I have migrated the 9x18mm and 5.45x39mm into their own files.

### CODE REVIEW

File: data/json/items/ammo/9x18.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id" : "9x18mm",
```
Comment (mugling) [inline data/json/items/ammo/9x18.json]: Style. Everything else in this folder uses the syntax without the leading space before the colon

**Comment by mugling** [comment]
Did you set item drops?

**Comment by BorkBorkGoesTheCode** [comment]
The 9x18 cartridge was already in the item drop list, but the 545 was not. I am going to put the 545 in the rifle_obscure list. Would you suggest a spawn rate of 100 700nx, 300 545x39, and 200 545 AP?

**Comment by mugling** [comment]
No, I'd go with 150 for `545x39` and `100` for the AP variant

**Comment by BorkBorkGoesTheCode** [comment]
How common is the trex gun supposed to be in the future? In the present a gun like the trex sells for +$250,000 US and the bullets are made to order at a price of +$100.

**Comment by BorkBorkGoesTheCode** [comment]
I have removed a large number of extra "m"s from ammo.

**Comment by mugling** [comment]
I'm not sure, however the `525x25` is going to be extremely rare in New England.

**Comment by BorkBorkGoesTheCode** [comment]
Would anyone object if I increase the spawn rate of the trex by 1?

How do the main.json entries interact with the gun entries?

**Comment by BorkBorkGoesTheCode** [comment]
I think it may be a good idea to create a foreign_soldiers item spawn list, but I won't add it here without mass approval.

**Comment by BorkBorkGoesTheCode** [comment]
What about 65 700nx, 150 545, 125 545AP, and +1 Elephant gun?

**Comment by mugling** [comment]
Whats +1 elephant gun? Try to keep each separate PR on topic. The earlier numbers I gave you should be suitable

**Comment by BorkBorkGoesTheCode** [comment]
A spawn rate increase of 1. main.json lists the elephant gun under rifles, with a spawn of 0.

**Comment by mugling** [comment]
Thats something different and can be ignored

**Comment by BorkBorkGoesTheCode** [comment]
Now the guns need to be added.

File: data/json/items/ammo/9x18.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  },
+  {   
+    "id": "9x18mmP2",
+    "copy-from": "9x18mm",
+    "type": "AMMO",
+    "name": "9x18mm SP-7",
+    "name_plural": "9x18mm SP-7",
+    "description": "9x18mm Makarov SP-7 ammunition.  Analogous to the western 9x19mm cartridge, the 9x18mm round was very common in the Eastern Bloc during the 20th century and remained in Russian military service into the 21st century.",
+    "weight": 8,
```
Comment (mugling) [inline data/json/items/ammo/9x18.json]: redundant

File: data/json/items/ammo/9x18.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    "effects": ["COOKOFF"]
+  },
+  {   
+    "id": "9x18mmfmj",
+    "copy-from": "9x18mm",
+    "type": "AMMO",
+    "name": "9x18mm RG028",
+    "name_plural": "9x18mm RG028",
+    "description": "9x18mm Makarov RG028 ammunition.  The RG028 round uses bullets with a hardened steel core to improve armor penetration.",
+    "weight": 8,
```
Comment (mugling) [inline data/json/items/ammo/9x18.json]: redundant

File: data/json/items/ammo/9x18.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id": "9x18mm",
```
Comment (mugling) [inline data/json/items/ammo/9x18.json]: consider copying from `9mm` to keep the two in sync

**Comment by BorkBorkGoesTheCode** [comment]
Scratch that.

File: data/json/items/ammo/9x18.json (hunk 2 above)
Comment (BorkBorkGoesTheCode) [inline data/json/items/ammo/9x18.json]: Do you want the 9x18mmP to copy from the 9x18mmfmj?

Comment (BorkBorkGoesTheCode) [inline data/json/items/ammo/9x18.json]: If the weight of 9x18 is a typo, I can remove the fields.

Comment (BorkBorkGoesTheCode) [inline data/json/items/ammo/9x18.json]: After researching the 9x18, it appears the weights listed are all inaccurate.

Comment (mugling) [inline data/json/items/ammo/9x18.json]: If your goal is migrating the remaining guns to use magazines then thats ok. However if it's re-balancing the entire guns/ammo then that's a job I can't help you with now and shouldn't be attempted immediately prior to `0.D`.

Although in no way are you at fault this piecemeal comment/reply approach is consuming a lot of time. Try and stay on-topic and push larger commits. You'll need to learn to rebase as opening multiple similar PR's isn't the best approach.

Comment (BorkBorkGoesTheCode) [inline data/json/items/ammo/9x18.json]: I was talking about the 9x18 cartridge weights.

File: data/json/items/ammo/9x18.json (hunk 4 above)
Comment (BorkBorkGoesTheCode) [inline data/json/items/ammo/9x18.json]: I'd rather keep the two separate.

**Comment by BorkBorkGoesTheCode** [comment]
I have no patience for the command line.

File: data/json/items/ammo/9x18.json (hunk 4 above)
Comment (mugling) [inline data/json/items/ammo/9x18.json]: `9x18` is almost identical to `9mm` and having the former defined in terms of the latter would help keep them in balance.

**Comment by karols146** [comment]
Git can be hard sometimes, but when you got how to do stuff is awesome.

**Pull request closed without merging**